lib/mal_import.rb

```diff
@@ -111,7 +111,7 @@ def metadata
         end
       rescue
       end,
-      poster_image: begin poster_image(@sidebar.css("img")[0]['data-src']) rescue nil end,
+      poster_image: begin poster_image(@sidebar.css("img")[0]['src']) rescue nil end,
```

Comment:
Didn't we actually make this change like 2 months ago when they started lazy-loading images?